aqua.yaml

```diff
@@ -8,14 +8,14 @@ registries:
     name: local
     path: registry.yaml
```

Comment:
Can you update the version of aqua-registry?
Then, you can remove `registry.yaml` and 3 lines above.